nimble/host/include/host/ble_gap.h

```diff
@@ -135,6 +135,7 @@ struct hci_conn_update;
 #define BLE_GAP_EVENT_PERIODIC_SYNC_LOST    22
 #define BLE_GAP_EVENT_SCAN_REQ_RCVD         23
 #define BLE_GAP_EVENT_PERIODIC_TRANSFER     24
+#define BLE_GAP_EVENT_PARING_FAILED         25
```

Comment:
Did you consider BLE_GAP_EVENT_PAIRING_COMPLETE with a status and reason?

Reply:
makes sense - I changed that and event is now sent also on success (`ble_sm_key_exch_success`)